### PR TITLE
[Fix-11217] [Doc] add postgresql config in doc: datasource-setting

### LINKED ISSUE #11217: [Doc][pseudo-cluster.md] typo and lack postgresql-initialization for install-guide
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

There is a typo and could be fix with issue #11216 together.

image

There is no initial guide for postgresql database, I think it needs a supply.

Standalone part and other way may have the same problem, need double check. 

### Documentation Links

https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/installation/pseudo-cluster.html

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ChrisYuan**
## Purpose of the pull request

fix #11217, improve the datasource-setting doc by adding postgresql.

## Brief change log

- fix a type '次为非重要信息' -> '此为非重要信息'
- add how to create database/user and give permission in postgresql 
- add postgresql datasource config example

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Review by SbloodyS — DISMISSED** [review]
LGTM

File: docs/docs/en/guide/howto/datasource-setting.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ export SPRING_DATASOURCE_PASSWORD={password}
 ## Pseudo-Cluster/Cluster Initialize the Database
 
 DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the libs directory of DolphinScheduler
-which is `api-server/libs/` and `alert-server/libs` and `master-server/libs` and `worker-server/libs`. Let's take MySQL as an example for how to initialize the database:
+which is `api-server/libs/` and `alert-server/libs` and `master-server/libs` and `worker-server/libs`, just ignore this steps if you use PostgreSQL. Let's walk through how to initialize the database in MySQL
```
Comment (zhongjiajie) [inline docs/docs/en/guide/howto/datasource-setting.md]: How about adding note syntax for mysql part?

```md
NOTE: If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the libs directory of DolphinScheduler ....
```

File: docs/docs/en/guide/howto/datasource-setting.md [hunk 2]
```diff
@@ -51,16 +52,44 @@ mysql> GRANT ALL PRIVILEGES ON dolphinscheduler.* TO '{user}'@'localhost';
...
 ``` 
 
-Then, modify `./bin/env/dolphinscheduler_env.sh` to use mysql, change {user} and {password} to what you set in the previous step.
+For PostgreSQL: 
+```shell
+# Use psql-tools to login PostgreSQL
+psql
+# Create a database
+postgres=# CREATE DATABASE dolphinscheduler;
+# Replace {user} and {password} with your username and password
```
Comment (zhongjiajie) [inline docs/docs/en/guide/howto/datasource-setting.md:65]: I think we should use `<>` for the required parameter, could you please also change the other place?
```suggestion
# Replace <user> and <password> with your username and password
```

Comment (ChrisYuan) [reply docs/docs/en/guide/howto/datasource-setting.md:61]: OK, got it.

File: docs/docs/en/guide/howto/datasource-setting.md (hunk 1 above)
Comment (ChrisYuan) [reply docs/docs/en/guide/howto/datasource-setting.md]: oh, I know, let me change it.

File: docs/docs/en/guide/howto/datasource-setting.md (hunk 2 above)
Comment (ChrisYuan) [reply docs/docs/en/guide/howto/datasource-setting.md:65]: Hi there, is it a bit strange for changing postresql part only? I think '{}' is clear for others, let's keep it or talk?

File: docs/docs/en/guide/howto/datasource-setting.md (hunk 1 above)
Comment (ChrisYuan) [reply docs/docs/en/guide/howto/datasource-setting.md]: changed, please recheck, thanks.

**Comment by ChrisYuan** [comment]
ping, please review, thanks~

File: docs/docs/en/guide/howto/datasource-setting.md [hunk 3]
```diff
@@ -19,8 +22,10 @@ export SPRING_DATASOURCE_PASSWORD={password}
 
 ## Pseudo-Cluster/Cluster Initialize the Database
 
-DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the libs directory of DolphinScheduler
-which is `api-server/libs/` and `alert-server/libs` and `master-server/libs` and `worker-server/libs` and `tools/libs`. Let's take MySQL as an example for how to initialize the database:
+DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. Let's walk through how to initialize the database in MySQL and PostgreSQL :
```
Comment (EricGao888) [inline docs/docs/en/guide/howto/datasource-setting.md]:
```suggestion
DolphinScheduler stores metadata in `relational database`. Currently, we support `PostgreSQL` and `MySQL`. Let's walk through how to initialize the database in `MySQL` and `PostgreSQL` :
```

File: docs/docs/zh/guide/howto/datasource-setting.md [hunk 4]
```diff
@@ -19,8 +21,10 @@ export SPRING_DATASOURCE_PASSWORD={password}
 
 ## 伪分布式/分布式安装初始化数据库
 
-DolphinScheduler 元数据存储在关系型数据库中，目前支持 PostgreSQL 和 MySQL，如果使用 MySQL 则需要手动下载 [mysql-connector-java 驱动][mysql] (8.0.16) 并移动到 DolphinScheduler 的每个模块的 libs 目录下
-其中包括 `api-server/libs/` 和 `alert-server/libs` 和 `master-server/libs` 和 `worker-server/libs` 和 `tools/libs`。下面以 MySQL 为例，说明如何初始化数据库
+DolphinScheduler 元数据存储在关系型数据库中，目前支持 PostgreSQL 和 MySQL。下面分别介绍如何使用 MySQL 和 PostgresQL 初始化数据库.
```
Comment (EricGao888) [inline docs/docs/zh/guide/howto/datasource-setting.md]:
```suggestion
DolphinScheduler 元数据存储在关系型数据库中，目前支持 PostgreSQL 和 MySQL。下面分别介绍如何使用 MySQL 和 PostgresQL 初始化数据库。
```

File: docs/docs/en/guide/howto/datasource-setting.md (hunk 3 above)
Comment (EricGao888) [inline docs/docs/en/guide/howto/datasource-setting.md]: In tech writing, we usually avoid `passive voice` unless necessary.

**Review by EricGao888 — COMMENTED** [review]
Except some nits, LGTM

File: docs/docs/en/guide/howto/datasource-setting.md (hunk 3 above)
Comment (ChrisYuan) [reply docs/docs/en/guide/howto/datasource-setting.md]: > In tech writing, we usually avoid `passive voice` unless necessary.

OK, thanks for your advice, keep this rule in the future.

**Review by EricGao888 — APPROVED** [review]

**Comment by EricGao888** [comment]
@ChrisYuan Good job! Thanks for your contributions!

**Pull request merged**